### ISSUE TITLE
DHCP spoofing seems to be failing on windows machines

### ISSUE THREAD

**Issue by clshack** [byt3bl33d3r/MITMf#64]
Hi, 
sorry for the subject but I think there some problem on the dhcp spoof module. 

```
python mitmf.py -i eth0 --spoof --dhcp --gateway 192.168.40.2  
```

On windows machine I launch:
```
ipconfig /release
ipconfig /renew
```

On the mitmf console I see these logs:
```
2015-04-15 17:31:54 Got DHCP DISCOVER from: 00:0c:***a:d1:cb xid: 0x47941a5c
2015-04-15 17:31:54 Sending DHCP OFFER
2015-04-15 17:31:54 Got DHCP REQUEST from: 00:0c***8a:d1:cb xid: 0x47941a5c
2015-04-15 17:31:54 Sending DHCP ACK
```

But the ip is not assigned to my windows machine. 

The same try on metasploit dhcp server works fine.

**Comment by byt3bl33d3r** [comment]
I'm currently looking into this, don't know exactly whats causing the issue yet but it seems to happen only with windows boxes.

**Comment by byt3bl33d3r** [comment]
@clshack this took way longer than it should , sorry about that :/ 
this is fixed in the upcoming 0.9.8 release which should be out by the end of the month

**Comment by byt3bl33d3r** [comment]
@clshack fixed in 0.9.8
